gulp/tasks/browserSync.js

```diff
@@ -1,7 +1,11 @@
-var gulp            = require('gulp'),
-    browserSync     = require('browser-sync'),
-    config          = require('../config').browserSync;
+var gulp = require('gulp'),
+    browserSync = require('browser-sync'),
+    config = require('../config').browserSync;
 
-gulp.task('browserSync', ['jekyll-build'], function() {
-    browserSync(config);
-});
+(function() {
```

Comment:
Nit: I think if you're moving the variables out you should probably remove the immediately invoked functions too. My understanding is that they are not necessary in node since it wraps each module inside a function itself.